ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -664,432 +642,369 @@ describe('WorkspaceEdit', () => {
+    await user.type(diseaseOfFocusInput, inValidInput);
+
+    await user.hover(saveButton);
+
+    expect(
+      screen.queryByText(/disease of focus cannot exceed 80 characters/i)
+    ).toBeInTheDocument();
   });
 
   it('should show error message if Other text for disseminate research is more than 100 characters', async () => {
-    const wrapper = component();
-    wrapper
-      .find('[data-test-id="OTHER-checkbox"]')
-      .at(1)
-      .simulate('change', { target: { checked: true } });
+    const errorMessage = /other methods of disseminating research findings/i;
+    renderComponent();
+
+    const otherDisseminationCheckbox = await screen.findByTestId(
+      'OTHER-checkbox'
+    );
+    await user.click(otherDisseminationCheckbox);
     const validInput = fp.repeat(8, 'a');
-    wrapper
-      .find('[data-test-id="otherDisseminateResearch-text"]')
-      .first()
-      .simulate('change', { target: { value: validInput } });
+    const otherDisseminateResearchText = screen.getByPlaceholderText(
+      OTHER_DISSEMINATION_REGEX
+    );
+    await user.type(otherDisseminateResearchText, validInput);
 
-    expect(
-      getSaveButtonDisableMsg(wrapper, 'otherDisseminateResearchFindings')
-    ).toBeUndefined();
-    const inValidInput = fp.repeat(101, 'b');
-    wrapper
-      .find('[data-test-id="otherDisseminateResearch-text"]')
-      .first()
-      .simulate('change', { target: { value: inValidInput } });
-    expect(
-      getSaveButtonDisableMsg(wrapper, 'otherDisseminateResearchFindings')
-    ).toBeDefined();
+    const saveButton = screen.getByRole('button', {
+      name: /create workspace/i,
+    });
+
+    await user.hover(saveButton);
+
+    expect(screen.queryByText(errorMessage)).not.toBeInTheDocument();
+
+    const invalidInput = fp.repeat(101, 'b');
+    await user.clear(otherDisseminateResearchText);
+    await user.type(otherDisseminateResearchText, invalidInput);
+    await user.hover(saveButton);
+
+    expect(screen.queryByText(errorMessage)).toBeInTheDocument();
   });
 
   it('should show error message if Other text for Special Population is more than 100 characters', async () => {
+    const errorMessage = /other specific population cannot/i;
     workspaceEditMode = WorkspaceEditMode.Edit;
-    const wrapper = component();
-    expect(
-      wrapper
-        .find('[data-test-id="specific-population-yes"]')
-        .first()
-        .prop('checked')
-    ).toEqual(true);
-    wrapper
-      .find('[data-test-id="other-specialPopulation-checkbox"]')
-      .at(1)
-      .simulate('change', { target: { checked: true } });
+    renderComponent();
 
-    const validInput = fp.repeat(100, 'a');
-    wrapper
-      .find('[data-test-id="other-specialPopulation-text"]')
-      .first()
-      .simulate('change', { target: { value: validInput } });
+    const specificPopulationYesRadio = await screen.findByTestId(
+      'specific-population-yes'
+    );
+    await user.click(specificPopulationYesRadio);
```

Comment:
is this right?  The old test looks like it's confirming that it's already checked.